apps/backoffice/src/app/api/services/route.ts

```diff
@@ -30,28 +30,30 @@ export const POST = withJWTAuthHandler(
       } catch (error: any) {
         return handleBadRequestErrorResponse(error.message);
       }
-      if (isAdmin(backofficeUser)) {
-        if (
-          servicePayload.metadata.group_id &&
-          !(await groupExists(
-            backofficeUser.institution.id,
-            servicePayload.metadata.group_id,
-          ))
-        ) {
-          return handleBadRequestErrorResponse(
-            "Provided group_id does not exists",
-          );
-        }
-      } else {
-        if (
-          servicePayload.metadata.group_id &&
-          !backofficeUser.permissions.selcGroups?.includes(
-            servicePayload.metadata.group_id,
-          )
-        ) {
-          return handleForbiddenErrorResponse(
-            "Cannot set service group relationship",
-          );
+      if (process.env.GROUP_AUTHZ_ENABLED?.toLowerCase()) {
```

Comment:
Consider to add this env var as **boolean** config parameter in `apps/backoffice/src/config/index.ts`

Reply:
fixed with 5e1d910e4a8e9dbe31da4dad1cc951635e69f5ec